backend/app/tests/test_series_list.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+import pytest
+from django.urls import reverse
+from django.test import Client
+
+from core.models import User, Bookmark
+
+client = Client()
+
+
+@pytest.mark.django_db
+def test_series_list_view_can_return_response():
```

Comment:
Decompose this test into multiple tests
- test_series_list_view_can_invalidate_unauthenticated_user
- test_series_list_view_can_invalidate_user_without_permission
- test_series_list_view_can_return_user_series
- test_series_list_view_can_return_empty_list_in_case_of_no_series